packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -68,6 +68,24 @@ export function getBlockchainByChainId(chain: number) {
     return chainIdMap.get(chain);
 }
 
+export function getChainIdByBlockchain(chain: Blockchain) {
+    return new Map([
+        [Blockchain.ETHEREUM, 1],
+        [Blockchain.POLYGON, 137],
+        [Blockchain.BSC, 56],
+        [Blockchain.OPTIMISM, 0], // TODO
+        [Blockchain.ARBITRUM, 42161],
+        [Blockchain.ARBITRUM_NOVA, 42170],
+        [Blockchain.ZORA, 7777777],
+        [Blockchain.SOLANA, 0], // TODO
+        [Blockchain.CARDANO, 0], // TODO
+        [Blockchain.GOERLI, 5],
+        [Blockchain.SEPOLIA, 0], // TODO
+        [Blockchain.MUMBAI, 80001],
+        [Blockchain.TESTNET, 0], // TODO
```

Comment:
Is this `Blockchain.TESTNET` a typo?

Reply:
I think it's the future localtesnet, but I can delete it and add it later